### DISCUSSION TITLE
How to slugify category name in permalink?

### DISCUSSION THREAD

**Discussion by liblaf** [mmistakes/minimal-mistakes#3580] [Q&A] [answered]
If there are spaces in category name, the permalink generated will contain something ugly like `%20`.
I found that `slugified_categories`, which wad introduced in `jekyll 4.1`, may solve the problem, but `github-pages` depends on `jekyll 3.9`.
Is there anyway to use `slugified_categories`?

**Comment by iBug** [comment, marked as answer]
Just use [GitHub Actions](https://jekyllrb.com/docs/continuous-integration/github-actions/) to deploy your site and you're no longer restricted to what GitHub Pages provides.

Also, this is not a theme-related question. Might be better to ask on Jekyll Talk.
